neverbleed.c

```diff
@@ -324,10 +343,33 @@ static int expbuf_write(struct expbuf_t *buf, int fd)
     return 0;
 }
 
+static void yield_on_data(int fd)
```

Comment:
I think it'd be a good idea to explain why this function exists. Maybe something like:

```suggestion
/**
 * This function waits for the provided socket to become readable, then calls `nanosleep(1)` before returning.
 * The intention behind sleep is to provide the application to complete its event loop before the neverbleed process starts
 * spending CPU cycles on the time-consuming RSA operation.
 */
static void yield_on_data(int fd)
```